### ISSUE TITLE
[Help] Uncought Permissions Forbidden error

### ISSUE THREAD

**Issue by Kowlin** [Cog-Creators/Red-DiscordBot#1850]
Following exception occurs when the bot cannot send a message to the user. when using ``help``

```Exception in command 'help'
Traceback (most recent call last):
  File "/bot/redbot/core/help_formatter.py", line 417, in help
    await destination.send(embed=embed)
  File "/root/.local/lib/python3.6/site-packages/discord/abc.py", line 769, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/root/.local/lib/python3.6/site-packages/discord/http.py", line 212, in request
    raise Forbidden(r, data)
discord.errors.Forbidden: FORBIDDEN (status code: 403): Cannot send messages to this user

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/bot/redbot/core/help_formatter.py", line 420, in help
    await destination.send(embed=embed)
  File "/root/.local/lib/python3.6/site-packages/discord/abc.py", line 769, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/root/.local/lib/python3.6/site-packages/discord/http.py", line 212, in request
    raise Forbidden(r, data)
discord.errors.Forbidden: FORBIDDEN (status code: 403): Cannot send messages to this user

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: Forbidden: FORBIDDEN (status code: 403): Cannot send messages to this user```

**Comment by tekulvw** [comment]
~~This is a 3rd party cog bug.~~

**Comment by Redjumpman** [comment]
I'm trying to reproduce this error and can't seem to do it. Can you list the steps necessary to reproduce it? I know that if you block the bot or disable DMs it should reply with `
I couldn't send the help message to you in DM. Either you blocked me or you disabled DMs in this server.`

**Comment by aikaterna** [comment]
Looks like I didn't see this, reported it in #1901 and then fixed it with #1955 .

**Comment by Tobotimus** [comment]
Duplicate of #1901 then (despite this one coming first, sorry Kowlin :D)
